includes/wikia/tests/GlobalTitleTest.php

```diff
@@ -48,47 +57,46 @@ function testNewFromText2() {
 		$this->assertTrue( $title->getText() === "Test Ze Spacjami", "Underscores, spaces expected" );
 	}
 
-	/**
-	 * @group UsingDB
-	 */
 	function testUrlsMainNS() {
 		$title = GlobalTitle::newFromText( "Timeline", NS_MAIN, 113 ); # memory-alpha
 		$url = "http://en.memory-alpha.org/wiki/Timeline";
 		$this->assertTrue( $title->getFullURL() === $url, sprintf("%s = %s, NOT MATCH", $title->getFullURL(), $url ) );
 	}
 
-	/**
-	 * @group UsingDB
-	 */
 	function testUrlsMainNSonWoW() {
 		$title = GlobalTitle::newFromText( "Main", 116, 490); # wowwiki
 		$url = "http://www.wowwiki.com/Portal:Main";
 		$this->assertTrue( $title->getFullURL() === $url, sprintf("%s = %s, NOT MATCH", $title->getFullURL(), $url ) );
```

Comment:
One more things - use ``$this->assertEquals`` instead to have a better assertion failed messages (instead of having to format them manually).